DataPlotly/layouts/plot_layout_item.py

```diff
@@ -138,6 +142,13 @@ def refresh(self):
         self.html_loaded = False
         self.invalidateCache()
 
+    def createExpressionContext(self):
```

Comment:
I think this should be removed --QgsLayoutItem has this function already implemented, and in a way which correctly handles atlas and layout context